go.mod

```diff
@@ -100,7 +100,7 @@ require (
 	github.com/siderolabs/siderolink v0.2.0
 	github.com/spf13/cobra v1.6.0
 	github.com/spf13/pflag v1.0.5
-	github.com/stretchr/testify v1.8.0
```

Comment:
there _should_ be more here, as at least AWS SDK should be bumped.

`make go-mod-outdated` lists everything

Reply:
should be good now

Reply:
I don't see any changes